tests/Support/Helper/DataType/TestDataHelper.php

```diff
@@ -850,7 +850,7 @@ public function assertUser(Concrete $object, string $field, int $seed = 1): void
         $this->assertEquals($expected, $value);
     }
 
-    public function assertVideo(Concrete $object, string $field, int $seed = 1, mixed $returnParams): void
+    public function assertVideo(Concrete $object, string $field, int $seed, mixed $returnParams): void
```

Comment:
or switch the required to the left? keeping it consistent. Just need to change the call here as well https://github.com/pimcore/pimcore/blob/68f198d32add713f2e4025b5dcb6414754133145/tests/Support/Test/DataType/AbstractDataTypeTestCase.php#L806

Reply:
Changed it https://github.com/pimcore/pimcore/pull/15820/commits/2ee1415138632fc88d5b3e20c44bc28caac85857